polaris-react/src/components/ActionMenu/components/SecondaryAction/SecondaryAction.scss

```diff
@@ -1,6 +1,5 @@
 @import '../../../../styles/common';
-
-$breakpoint: 768px;
+@import '../../../../styles/media-queries';
```

Comment:
Should this just be added to `common` so we don't need to import it everywhere?

Reply:
Works for me! Think we just need to update the output directory in `gen-media-queries.js` and add it to `src/styles/common` 👍

Reply:
We ended up adding the export to `common`!